examples/README.md

```diff
@@ -0,0 +1,193 @@
+# Examples
+
+This directory contains comprehensive examples demonstrating how to use the MIP SDK with MicroStrain inertial 
+sensors. The examples are organized by device series and programming language, providing both C and C++ 
+implementations for most use cases.
+
+## MicroStrain Device Overview
```

Comment:
Before this, it might be nice to have a "Quick Start" section that just instructs users to clone the whole repository and build with `-DBUILD_EXAMPLES=ON` as otherwise it requires a lot of reading and clicking around for users to eventually figure out how to actually build and run an example. You could then link to the "Getting Started" section below to talk about exactly what to do after you have the project built or maybe just lump that in with this section, and then mention that more device specific documentation can be found in the individual READMEs